yt_dlp/extractor/netverse.py

```diff
@@ -0,0 +1,212 @@
+            'display_id': display_id,
+            'title': videos.get('title'),
+            'season': videos.get('season_name'),
+            'thumbnail': traverse_obj(videos, ('program_detail', 'thumbnail_image')),
+            'description': traverse_obj(videos, ('program_detail', 'description')),
+            'episode_number': videos.get('episode_order'),
+            'series': traverse_obj(videos, ('program_detail', 'title')),
+        }
+
+
+class NetversePlaylistIE(NetverseBaseIE):
+    _VALID_URL = r'https?://(?:\w+\.)?netverse\.id/(?P<type>webseries)/(?P<display_id>[^/?#&]+)'
+    _TESTS = [{
+        # multiple season
+        'url': 'https://netverse.id/webseries/tetangga-masa-gitu',
+        'info_dict': {
+            'id': 'tetangga-masa-gitu',
+            'title': 'Tetangga Masa Gitu',
+        },
+        'playlist_count': 519,
+    }, {
+        # single season
+        'url': 'https://netverse.id/webseries/kelas-internasional',
+        'info_dict': {
+            'id': 'kelas-internasional',
+            'title': 'Kelas Internasional',
+        },
+        'playlist_count': 158,
+    }]
+
+    def parse_single_season_playlist(self, input_data, page_num, custom_id=None, season_id='',
+                                     force_endpoint_type='auto', input_type='url'):
+
+        _, playlist_json = self._call_api(
+            input_data, query={'page': page_num + 1}, season=season_id, custom_id=custom_id,
+            force_endpoint_type=force_endpoint_type, input_type=input_type)
+        for slug in traverse_obj(playlist_json, ('response', ..., 'data', ..., 'slug')):
+            yield self.url_result(f'https://www.netverse.id/video/{slug}', NetverseIE)
+
+    def parse_playlist(self, url, json_data, playlist_id):
+        slug_sample = traverse_obj(json_data, ('related', 'data', ..., 'slug'))[0]
+        season_id_list = [season.get('id') for season in json_data.get('seasons')]
+
+        for season in season_id_list:
+            # initial data
+            _, playlist_json = self._call_api(
+                input_data=slug_sample, custom_id=playlist_id, season=season,
+                force_endpoint_type='season', input_type='slug')
+
+            season_list = traverse_obj(playlist_json, ('response', 'season_list'))
+            number_video_per_page = season_list.get('to') - season_list.get('from') + 1
+            number_of_pages = season_list.get('last_page')
+
+            yield from InAdvancePagedList(
+                functools.partial(
+                    self.parse_single_season_playlist, slug_sample, custom_id=playlist_id,
+                    season_id=season, force_endpoint_type='season', input_type='slug'),
+                number_of_pages, number_video_per_page)
+
+    def _real_extract(self, url):
+        _, playlist_data = self._call_api(url)
+        return self.playlist_result(
+            self.parse_playlist(url, playlist_data['response'], _),
```

Comment:
The point of `_` is that it is unused. If you want it, give it a name!

Reply:
Fixed